src/TSMapEditor/Rendering/MapView.cs

```diff
@@ -2021,5 +2041,14 @@ private void InstantRenderMinimap()
 
             EditorState.RenderInvisibleInGameObjects = true;
         }
+
+        private Point2D GetRelativeTilePositionFromCursorPosition(MapTile tile)
+        {
+            var tileCenter = tile.GetTileCenter();
+            var cursorPosition = GetCursorMapPoint() + new Point2D(0, Constants.MapYBaseline);
+            var offset = cursorPosition - CellMath.CellCenterPointFromCellCoords(tile.CoordsToPoint(), Map);
+
+            return tileCenter + offset;
+        }
```

Comment:
This can be simplified, just like `MapTile.GetTileCenter`. I believe the following should work.

```suggestion
        private Point2D GetRelativeTilePositionFromCursorPosition(MapTile tile)
        {
            return GetCursorMapPoint() - CellMath.CellTopLeftPointFromCellCoords_NoBaseline(tile.CoordsToPoint(), Map);
        }
```

However for TS and RA2/YR support you'll need to also check if `!Constants.IsFlatWorld && !EditorState.Is2DMode`, and if so, call `CellTopLeftPointFromCellCoords_3D_NoBaseline` instead.